services/submission/src/test/java/app/coronawarn/server/services/submission/controller/SubmissionControllerTest.java

```diff
@@ -539,7 +539,8 @@ void testValidCheckinData() {
 
     final ResponseEntity<Void> actResponse = executor.executePost(buildPayloadWithCheckinData(invalidCheckinData));
     assertThat(actResponse.getStatusCode()).isEqualTo(OK);
-    assertTraceWarningsHaveBeenSaved(2);
+    assertTraceWarningsHaveBeenSaved(invalidCheckinData.size()
+        + invalidCheckinData.size() * config.getRandomCheckinsPaddingMultiplier());
```

Comment:
Shouldn't invalid checkin data be filtered out?

Reply:
👍